Filters/exclusions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdGuardDNS/issues/555
```

Comment:
It seems that this links doesn't work, I guess it should be `https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/555`

Reply:
fixed